python/aitemplate/backend/rocm/gemm/gemm_rcr_bias_fast_gelu.py

```diff
@@ -26,6 +26,7 @@
 
 
 @registry.reg("rocm.gemm_rcr_bias_fast_gelu.config")
+@registry.reg("rocm.gemm_rcr_bias_gelu.config")
```

Comment:
@fsx950223 This file says "fast_gelu", should registry for "gelu" be in this file?

Reply:
Yes, fast_gelu is alias of gelu in ROCM.